tests/helper.py

```diff
@@ -1,44 +1,53 @@
 import os
 import tempfile
+from pathlib import Path
 
 from facefusion.filesystem import create_directory, is_directory, is_file, remove_directory
 from facefusion.typing import JobStatus
 
 
-def is_test_job_file(file_path : str, job_status : JobStatus) -> bool:
-	return is_file(get_test_job_file(file_path, job_status))
+# Get the base temporary directory once and reuse it
+TEMP_DIR = Path(tempfile.gettempdir())
 
+def is_test_job_file(file_path: str, job_status: JobStatus) -> bool:
+    return is_file(get_test_job_file(file_path, job_status))
 
-def get_test_job_file(file_path : str, job_status : JobStatus) -> str:
-	return os.path.join(get_test_jobs_directory(), job_status, file_path)
 
+def get_test_job_file(file_path: str, job_status: JobStatus) -> str:
+    return str(get_test_jobs_directory() / job_status / file_path)
 
-def get_test_jobs_directory() -> str:
-	return os.path.join(tempfile.gettempdir(), 'facefusion-test-jobs')
 
+def get_test_jobs_directory() -> Path:
```

Comment:
I actual like to have Path as return type, but I would prefer joinpath() everywhere